Effects/LCAS.hlsl

```diff
@@ -0,0 +1,69 @@
+// This is a combination of linear interpolation and light version of cas
+
+//!MAGPIE EFFECT
+//!VERSION 1
+

```

Comment:
https://github.com/Blinue/Magpie/blob/e552e60621e6acbf9d87fec1770e797dcf74f546/Effects/CAS.hlsl#L5-L6

Two lines should be added here to indicate that this effect does not support scaling (the scale parameter is not supported)